task1/task1.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 1</title>
+</head>
+
+<body>
+    <table border=1 cellspacing=0 cellpadding=5px>
+        <thead>
+            <tr height=100px>
+                <th colspan="3">
+                    <font size="6">Markup of “Basic HTML” task</font>
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr valign=top>
+                <td width=200px>
```

Comment:
does not work. Take a look at `col` and `colgroup` please
![image](https://user-images.githubusercontent.com/11594423/74338191-ff81ee00-4da1-11ea-8d56-c1b300a20cf2.png)

Reply:
Does not work, because in mockup you requested a padding of 5px and a table with border (which in this case has width 1px), I will correct td "width" attribute to respect the border and the padding